src/header/value.rs

```diff
@@ -57,12 +57,16 @@ impl HeaderValue {
     /// assert_eq!(val, "hello");
     /// ```
     #[inline]
-    pub fn from_static(src: &'static str) -> HeaderValue {
+    #[allow(unconditional_panic)] // required for the panic circumventon
+    #[track_caller]
```

Comment:
Why are we adding `track_caller`?

Reply:
I _believe_ this will provide a more useful line number when the panic occurs. Instead of pointing at hyper's code, it will point at the user's `HeaderValue::from_static` line

Reply:
Does removing this attribute make the panic message much worse?

Reply:
Just tried without and it doesn't seem to change anything. Removed!